### PR TITLE
September v3 Release Notes 

### PR DESCRIPTION
**Pull request by yihuiliao**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

### CODE REVIEW

File: scripts/changelog.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const octokit = new Octokit();
 run();
 
 async function run() {
-  let packages = JSON.parse(exec('yarn workspaces info --json').toString().split('\n').slice(1, -2).join('\n'));
+  let packages = exec('yarn workspaces list --json').toString().split(require('os').EOL).filter(Boolean).map((x) => JSON.parse(x));
```
Comment (yihuiliao) [inline scripts/changelog.js:11]: maybe there's a better way to do this but basically `yarn workspaces info --json` doesn't exist after we upgraded to yarn 4 hence the change

Comment (snowystinger) [reply scripts/changelog.js:11]: for now it's fine, though eventually we should turn it into a yarn plugin, then we don't have to do an exec call at all, we can use their workspaces API

we had to update it here as well when we switched to yarn 4

File: packages/dev/docs/pages/releases/2024-09-30.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+
+description: Happy fall! We’re excited to announce that our color picker components are now generally available! This includes ColorPicker, ColorArea, ColorField, ColorSlider, ColorSwatch, ColorSwatchPicker, and ColorWheel. These enable you to build fully customizable color pickers, including accessible color descriptions for screen reader support.
+In addition, we’ve introduced a new pending state for buttons in React Aria Components, allowing you to display a progress spinner to indicate that an action is in progress. It also handles screen reader announcements and disables other interactions, except for focus, during the pending state.
+We're also advancing the React Aria Components Tree and Spectrum TreeView to beta, and have added some basic documentation. In alpha, we have new Disclosure and DisclosureGroup components in React Aria Components, which can be used to build accordions and other collapsible panels.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2024-09-30.mdx:17]: Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! We've also added a pending state for buttons in React Aria Components, advanced TreeView to beta, and added new Disclosure and DisclosureGroup components in alpha for building accordions and collapsible panels.

File: packages/dev/docs/pages/releases/2024-09-30.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+date: 2024-09-30
+---
+# September 30, 2024 Release
+Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! This includes ColorPicker, ColorArea, ColorField, ColorSlider, ColorSwatch, ColorSwatchPicker, and ColorWheel. These enable you to build fully customizable color pickers, including accessible color descriptions for screen reader support.
+In addition, we’ve introduced a new pending state for buttons in React Aria Components, allowing you to display a progress spinner to indicate that an action is in progress. It also handles screen reader announcements and disables other interactions, except for focus, during the pending state.
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2024-09-30.mdx:23]: Pretty sure the paragraphs also need space between them

File: packages/dev/docs/pages/releases/2024-09-30.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+---
+
+description: Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! We've also added a pending state for buttons in React Aria Components, advanced TreeView to beta, and added new Disclosure and DisclosureGroup components in alpha for building accordions and collapsible panels.
+
+date: 2024-09-30
+---
+
+# September 30, 2024 Release
+Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! This includes [ColorPicker](https://react-spectrum.adobe.com/react-aria/ColorPicker.html), [ColorArea](https://react-spectrum.adobe.com/react-aria/ColorArea.html), [ColorField](https://react-spectrum.adobe.com/react-aria/ColorField.html), [ColorSlider](https://react-spectrum.adobe.com/react-aria/ColorSlider.html), [ColorSwatch](https://react-spectrum.adobe.com/react-aria/ColorSwatch.html), [ColorSwatchPicker](https://react-spectrum.adobe.com/react-aria/ColorSwatchPicker.html), and [ColorWheel](https://react-spectrum.adobe.com/react-aria/ColorWheel.html). These enable you to build fully customizable color pickers, including accessible color descriptions for screen reader support.
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2024-09-30.mdx:21]:
```suggestion
Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! This includes [ColorPicker](https://react-spectrum.adobe.com/react-aria/ColorPicker.html), [ColorArea](https://react-spectrum.adobe.com/react-aria/ColorArea.html), [ColorField](https://react-spectrum.adobe.com/react-aria/ColorField.html), [ColorSlider](https://react-spectrum.adobe.com/react-aria/ColorSlider.html), [ColorSwatch](https://react-spectrum.adobe.com/react-aria/ColorSwatch.html), [ColorSwatchPicker](https://react-spectrum.adobe.com/react-aria/ColorSwatchPicker.html), and [ColorWheel](https://react-spectrum.adobe.com/react-aria/ColorWheel.html). These enable you to build fully customizable color pickers, and include accessible color descriptions for screen reader support.
```

File: packages/dev/docs/pages/releases/2024-09-30.mdx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+In addition, we’ve introduced a new [pending state](https://react-spectrum.adobe.com/react-aria/Button.html#pending) for buttons in React Aria Components, allowing you to display a progress spinner to indicate that an action is in progress. It also handles screen reader announcements and disables other interactions, except for focus, during the pending state.
+
+We're also advancing the React Aria Components [Tree](https://react-spectrum.adobe.com/react-aria/Tree.html) and Spectrum [TreeView](https://react-spectrum.adobe.com/react-spectrum/TreeView.html) to beta, and have added some basic documentation. In alpha, we have new [Disclosure](https://react-spectrum.adobe.com/react-aria/Disclosure.html) and [DisclosureGroup](https://react-spectrum.adobe.com/react-aria/DisclosureGroup.html) components in React Aria Components, which can be used to build accordions and other collapsible panels.
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2024-09-30.mdx:25]:
```suggestion
React Aria Components [Tree](https://react-spectrum.adobe.com/react-aria/Tree.html) and Spectrum [TreeView](https://react-spectrum.adobe.com/react-spectrum/TreeView.html) are moving to beta and we've started their documentation. We are introducing the new [Disclosure](https://react-spectrum.adobe.com/react-aria/Disclosure.html) and [DisclosureGroup](https://react-spectrum.adobe.com/react-aria/DisclosureGroup.html) components in React Aria Components. These can be used to build accordions and other collapsible panels.
```

File: packages/dev/docs/pages/releases/2024-09-30.mdx (hunk 4 above)
Comment (devongovett) [reply packages/dev/docs/pages/releases/2024-09-30.mdx:21]: they don't allow you to include accessible descriptions, they do include them. 😄 maybe "which include" 🤷

File: packages/dev/docs/pages/releases/2024-09-30.mdx (hunk 5 above)
Comment (devongovett) [reply packages/dev/docs/pages/releases/2024-09-30.mdx:25]: I prefer the original 🤷

Comment (snowystinger) [reply packages/dev/docs/pages/releases/2024-09-30.mdx:25]: I don't have a strong preference. I was mostly trying to change "In alpha, we have" which sounded too different from "We're also advancing". The sentence structures just don't match up for me.

File: packages/dev/docs/pages/releases/2024-09-30.mdx (hunk 4 above)
Comment (snowystinger) [reply packages/dev/docs/pages/releases/2024-09-30.mdx:21]: How about:

These components include accessible color descriptions for screen reader support and enable you to build fully customizable color pickers.

**Review by snowystinger — APPROVED** [review]
only a little sad that there is no line about happy fall and to celebrate the colors of fall, we have a color picker

**Comment by devongovett** [comment]
> only a little sad that there is no line about happy fall and to celebrate the colors of fall, we have a color picker

there's still time! 😅

**Pull request merged**